packages/babel-core/src/config/files/configuration.ts

```diff
@@ -263,17 +269,19 @@ const readConfigJSON5 = makeStaticFileCache((filepath, content): ConfigFile => {
   try {
     options = json5.parse(content);
   } catch (err) {
-    err.message = `${filepath}: Error while parsing config - ${err.message}`;
-    throw err;
+    throw new ConfigError(
+      `Error while parsing config - ${err.message}`,
+      filepath,
+    );
   }
 
-  if (!options) throw new Error(`${filepath}: No config detected`);
+  if (!options) throw new ConfigError(`No config detected`, filepath);
```

Comment:
Maybe there is some lint rule to make it a normal string, but in this PR I think it's ok.🤫

Reply:
There's [@typescript-eslint/quotes](https://typescript-eslint.io/rules/quotes), which can be configured with `allowTemplateLiterals: false` to avoid unnecessary template literals (despite the name, it does not forbid all usage of template literals). To be [compatible with Prettier](https://github.com/prettier/eslint-config-prettier#quotes), `avoidEscape: true` must also be set (even if you extend `eslint-config-prettier`, individual rule configurations take precedence).